### DISCUSSION TITLE
RTSP Resolution?

### DISCUSSION THREAD

**Discussion by myromeo** [roleoroleo/yi-hack-MStar#475] [Q&A] [answered]
Can anyone advise the rstp streams (high and low) resolution and frame rates? Integrating with Frigate NVR and this info would be really useful to optimise CPU usage!

Thanks in advance!

**Comment by roleoroleo** [comment, marked as answer]
high: 1920x1080@20fps
low: 640x360@20fps

**Comment by myromeo** [comment]
Thanks! Is it possible to lower the frame rates in camera?

**Reply by roleoroleo** [reply]
No. It's not possible, it depends on yi software.
